src/components/HomePage/HomePage.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './styles.less';
+import AddCompanies from 'containers/AddCompanies';
+import ListCompanies from 'containers/ListCompanies';
+import AddEmployee from '../AddEmployee';
```

Comment:
the reason you weren't seeing your props was two things. In your component you weren't accessing `this.props.addCompany`, and also you weren't using the container which had `addCompany` and was passing down to an instance of the component `AddCompanies`. Literally EVERYONE makes this mistake, and it's a `F**K!` kind of reaction every time... I just know all the error messages now because I've been screwing up for so long.